arelle/plugin/validate/XFsyntax/xf.py

```diff
@@ -80,7 +80,7 @@ def __init__(
         )
 
     @tatsumasu()
-    def _module_(self):  # noqa
```

Comment:
All of these noqa yielded errors like this:
`arelle/plugin/validate/XFsyntax/xf.py:1907:28: NQA101 "# noqa" has no violations`